app/templates/components/forms/admin/content/social-links-form.hbs

```diff
@@ -61,5 +61,21 @@
       @isSocialLink={{true}}
       @inputId="patreon" />
   </div>
+  <div class="field">
+    <label>{{t 'Gitter'}}</label>
+    <Widgets::Forms::LinkInput
+      @segmentedLink={{this.socials.segmentedGitterUrl}}
```

Comment:
This does not exist

Reply:
What do you mean?
Is it not appearing on the front-end ?

Reply:
segmentedGitterUrl does not exist